simulator/dummy_perception_publisher/include/dummy_perception_publisher/node.hpp

```diff
@@ -116,6 +116,7 @@ class DummyPerceptionPublisherNode : public rclcpp::Node
   bool enable_ray_tracing_;
   bool use_object_recognition_;
   bool use_real_param_;
+  bool use_baselink_z_;
```

Comment:
```suggestion
  bool use_base_link_z_;
```
I would like to use base_link to be consistent with others.